airbyte-webapp/src/components/connection/CatalogTree/next/NextTable/NextTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { ColumnDef, flexRender, useReactTable, getCoreRowModel } from "@tanstack/react-table";
+import classNames from "classnames";
+import { PropsWithChildren } from "react";
+
+import styles from "./NextTable.module.scss";
+
+export interface TableProps<TData> {
+  data: TData[];
+  columns: Array<ColumnDef<TData>>;
```

Comment:
```suggestion
  columns: ColumnDef<TData>[];
```

Reply:
We went back and forth about this rule and this how we decided the linter would handle these situations. Should be the result of auto-fixing.